tests/js/server/dump/dump-db-properties-setup-cluster-single.js

```diff
@@ -0,0 +1,54 @@
+/*jshint globalstrict:false, strict:false, maxlen : 4000 */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief setup collections for dump/reload tests
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2019 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Jan Christoph Uhde
+/// @author Copyright 2019, ArangoDB GmbH, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+const db = require("@arangodb").db;
+
+function create(name, minReplicationFactor, replicationFactor, sharding){
+  try {
+    db._dropDatabase(name + "Src");
+  } catch (err1) {
+  }
+  try {
+    db._dropDatabase(name + "Dst");
+  } catch (err1) {
+  }
+  db._createDatabase(name + "Src", {
+    "minReplicationFactor": minReplicationFactor,
+    "replicationFactor": replicationFactor,
+    "sharding": sharding,
+  });
+}
+
+const doIt = create;
+
+doIt("UnitTestsDump", 2, 3, "");
```

Comment:
this is all a bit bloated. the main scope calls `doit`, which is an alias of `create`, which is a function that is called once.
Why not put the function body into the main scope? That would reduce code size by about 50%.